byron/chain/formal-spec/blockchain-spec.tex

```diff
@@ -754,6 +769,51 @@ \section{Epoch transitions}
 \label{fig:rules:epoch}
 \end{figure}
 
+\clearpage
+
+\section{Protocol parameters}
+
+The rules presented henceforth make use of a protocol parameters map, which map
```

Comment:
Shouldn't this be "which maps different [...]"? I guess the second part of the sentence is referring to the map specifically, and not rules.

Reply:
Ups, I modified this, but I forgot to push my changes :/